dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -230,7 +230,7 @@ def __init__(self,
             with p._check_api_type_scope(types.PipelineAPIType.ITERATOR):
                 p.schedule_run()
         self._first_batch = None
-        self._first_batch = self.next()
+        self._first_batch = DALIGenericIterator.next(self)
```

Comment:
```suggestion
        self._first_batch = DALIGenericIterator.__next__(self)
```
Because:
```
    def next(self):
        """
        Returns the next batch of data.
        """
        return self.__next__()
```